cli/azd/pkg/tools/azcli/azcli_test.go

```diff
@@ -204,68 +204,6 @@ func runAndCaptureUserAgent(t *testing.T) string {
 	return matches[0][1]
 }
 
-func Test_extractDeploymentError(t *testing.T) {
```

Comment:
do we have a test to validate the output looks like we expect for errors?

Reply:
Yes, AzureDeploymentError errors has a suite of tests.